README.md

```diff
@@ -0,0 +1,8 @@
+# File Content Extraction
+ The connector uses Apache Tika to extract content and artifacts from almost any file
+## Prerequisites
+The sample playbooks use filePreview field within the Indicators module to display theextracted data, you need to create the field first. a config export is located at:
```

Comment:
let's create a solution pack with the module definition and specifying this connector as a depdency

Reply:
keeping it inside this file-content-extraction will also bundle it into the connector rpm everytime and increase its size.

Reply:
Sure, the content pack could have a case or two of data extraction from a file, since this connector is a dependency i assume it has to be released first